src/common/core-functions.k

```diff
@@ -1,7 +1,37 @@
 /*@ \section{Module CORE-FUNCTIONS} */
 module CORE-FUNCTIONS
     imports CORE-SORTS
-    imports CORE-CLASSES
+    imports AUX-STRINGS
+
+//@ \subsection{Shortcuts for the most frequently used classes within the semantics}
+
+syntax KItem ::= "classObject"
```

Comment:
They don't have to be, I moved this part from another file to fix circulate imports, I didn't make an effort this time for trying to change it. I will do that seperately.